shell/src/main/java/alluxio/cli/fsadmin/journal/QuorumElectCommand.java

```diff
@@ -84,8 +84,8 @@ public int run(CommandLine cl) throws IOException {
       mPrintStream.println(String.format(OUTPUT_FAIL, serverAddress));
       return 0;
     }
-
     mPrintStream.println(String.format(OUTPUT_SUCCESS, serverAddress));
+    jmClient.resetPriorities(priorities.getPrioritiesList());
```

Comment:
What if this fails?

Reply:
added a `try / catch`
I call `resetPriorities` independently of `transferLeadership` success because the priorities could have been changed while the transfer failed.